usr/sbin/truenas-install

```diff
@@ -404,8 +422,14 @@ partition_disks()
 
     _disksparts=$(for _disk in ${_disks}; do
 	create_partitions ${_disk} >&2
-	echo /dev/${_disk}3
-    done)
+    # Check if nvme or non-nvme disk.
+    if [[ "${_disk}" == *"nvme"* ]]; then
```

Comment:
Also what do you think of having a function which returns the partition based on disk ?
The if block is same as above ?

Reply:
Yeah I can do that, I just have to figure out how, I'm refreshing all my memory doing this, I'm more familiar with languages like Dart and Java and I haven't touched them in forever. I'm thinking of having a function take a parameter or whatever it's called? But I don't see any of these functions in this file declaring parameters to take in? I'm gonna google it anyway though. Just give me a little and I'll recommit with it.

Reply:
No worries, please take your time, I can help with the patch as required then ; )

Reply:
I started over from the new copy of the file and made a function that I call to label the partitions appropriately. Is it looking better?